### PR TITLE
Add Tarides

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,4 @@ script/generate_opam_update_list
 opam_update_list
 *.bz2
 *.ai
+_opam
```
Comment (Chris00) [inline .gitignore:35]: ?

File: site/learn/companies.md [hunk 2]
```diff
@@ -558,6 +558,26 @@
...
     </dd>
 </dl>
+<dl class="row">
+    <dt class="span3">
+        <img src="/img/users/tarides.png" alt="Tarides">
+    </dt>
+    <dd class="span5">
+        <h2><a href="https://tarides.com">Tarides</a>, France</h2>
+        <p>We are building and maintaining open-source infrastructure tools in OCaml:
+        (1) <a href="https://mirage.io">MirageOS</a>, the most advanced unikernel project,
```
Comment (Chris00) [inline site/learn/companies.md:568]: You are conscious that it will be rendered as a continuous paragraph (not as an enumerated list), right?

Comment (samoht) [reply site/learn/companies.md:568]: yes; I've tried to use proper lists but that didn't render very well with the current css (too much space between lists).

File: .gitignore (hunk 1 above)
Comment (samoht) [reply .gitignore:35]: that's where opam2 store its local switches

**Comment by Chris00** [comment]
Thanks.

**Pull request merged**